### PR TITLE
Added Linux support

### PR DESCRIPTION
**Pull request by viggo-w**
Added Linux support by adding checks when assigning variables with paths. If in windows, use paths with backslash, else use paths with forward slash /

### CODE REVIEW

**Comment by SerjSX** [comment]
Hi! Thanks! Lol I literary changed \ to / today because I put the bs4 and customtkinter folders in zip files, so the program extracts them and on Github it doesn't show other types other than Python (it extracts if their folders aren't already there), was having issues with \b so then I changed all of them to /. That's why I won't merge the request, but stay with me because I still want you to test if it works and we'll work on it here together, I'm not on a Linux distribution so I can't.

**Comment by SerjSX** [comment]
I pushed the latest updates (not in Releases yet), please check it and test on your Linux machine, and lmk if all goes well. Can you also send a screenshot of how the GUI looks? I want to see it from a Linux distribution

File: pSearch.py [hunk 1]
```diff
@@ -29,32 +29,46 @@
 
 root = customtkinter.CTk()
 root.title("pSearch")
-root.iconbitmap(path + "\media\icon.ico")
+
```
Comment (SerjSX) [inline pSearch.py]: This line is approved and I will add a fix for it, thanks for mentioning

**Review by SerjSX — COMMENTED** [review]
This issue has been already fixed before I push it to the repository, coincidentally! But will continue testing with the author because I don't have access to a Linux distribution.

Changes wanted:
- \ to /  [Already done]
- No icon on Linux [Will do]

File: pSearch.py (hunk 1 above)
Comment (SerjSX) [reply pSearch.py]: (in case not clear it's the line of the icon, let me know if there's another file type used on linux)

**Comment by viggo-w** [comment]
Another thing: on linux, path and file names are case sensitive so we must change /media/github/GitHub-Mark-Light-32px.png to /media/Github/Git
Hub-Mark-Light-32px.png. other than that, it works great on debian.
2023-01-02-133600_1920x1080_scrot

**Comment by SerjSX** [comment]
> Another thing: on linux, path and file names are case sensitive so we must change /media/github/GitHub-Mark-Light-32px.png to /media/Github/Git Hub-Mark-Light-32px.png. other than that, it works great on debian. 2023-01-02-133600_1920x1080_scrot

Done! I haven't noticed that cause the other folders (others, media) are lowercase so I wrote that like that as well lol. Right now I pushed these changes (I posted this comment after committing them):
1. Icon only on Windows (if there's another file extension lmk)
2. github to Github

I noticed at the top there isn't closing button, is that from the OS or the icon error messed it up?
Let me know how it goes before I make a release.

**Comment by viggo-w** [comment]
> I noticed at the top there isn't closing button, is that from the OS or the icon error messed it up?
> Let me know how it goes before I make a release.

Thats because I use bsp window manager which does not have close buttons.

**Comment by SerjSX** [comment]
> Thats because I use bsp window manager which does not have close buttons.

Oh alright, lmk how it goes then generally and if there are other issues on Linux

**Pull request closed without merging**